plugins/UserPlugin/src/opengrok/auth/plugin/decoders/FakeOSSOHeaderDecoder.java

```diff
@@ -23,20 +23,22 @@
 package opengrok.auth.plugin.decoders;
 
 /**
- * Almost like @{code OSSOHeaderDecoder} however uses HTTP headers with
- * the "my-" prefix which allows for custom header insertion.
+ * Almost like @{code OSSOHeaderDecoder} however uses OSSO HTTP headers with
+ * prefix which allows for custom header insertion.
  * This class should therefore only be used for debugging.
  *
  * @author Krystof Tulinger
  */
 public class FakeOSSOHeaderDecoder extends OSSOHeaderDecoder {
 
+    private final String PREFIX = "fake-";
+    
     public FakeOSSOHeaderDecoder() {
-        OSSO_COOKIE_TIMESTAMP_HEADER = "my-osso-cookie-timestamp";
```

Comment:
This is not important now but maybe this should be called just SSO and not OSSO.

Reply:
I think these are specific to Oracle SSO so I'd leave them as they are.